### PR TITLE
Update ollama.js

### PR DESCRIPTION
**Pull request by qiwi1272**
Removed a todo, fixed typo.

### CODE REVIEW

**Review by DavidAJohnston — COMMENTED** [review]
Looks good to me. @betterbrand want to take a look?

File: src/service/ollama/ollama.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const fs = require("fs");
 const os = require("os");
 const path = require("path");
 const { exec } = require("child_process");
+const ollamaHost = process.env.OLLAMA_HOST;
```
Comment (shamoons) [inline src/service/ollama/ollama.js:5]: Do we have a .env file somewhere?

Comment (Qiwi2681) [reply src/service/ollama/ollama.js:5]: If we do, I missed it.

**Comment by betterbrand** [comment]
This has already been changed in the agent-merge branch that we showed yesterday. This would conflict. If needed after that branch is merged, a new PR should be created

**Comment by DavidAJohnston** [comment]
Thanks @betterbrand

**Pull request closed without merging**